### ISSUE TITLE
FE: Underline CTA links in Spotlight sections

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#13808]
## Description
On Vet centers and VAMC Health service lists, there are spotlight sections with CTAs.  To be more accessible and to be in line with the [VA design and style guidelines](https://design.va.gov/components/link/), these links should be underlined.

### Screenshots
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/1bd4d7e7-8f4a-44d4-b883-5550584340df)
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/95267c00-cf77-4e5a-a69e-01ebdf34cb47)

## Acceptance Criteria
- [ ] CTA links on Vet center spotlights are underlined
- [ ] CTA links on  VAMC Health service spotlights are underlined

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [x] `Facilities`
- [ ] `User support`

**Comment by kmariepat-cityfriends** [comment]
This AC: `-I expect if there is a design system component available to use that component (maybe its the card component) ` has been removed and will instead be addressed on

**Comment by jilladams** [comment]
Changes are deployed through Staging so far, will hit prod tomorrow. Verified in Staging: 
https://staging.va.gov/los-angeles-vet-center/
![Screenshot 2023-06-20 at 2 52 58 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/85581471/fdc6ddc5-8426-48a5-a630-bbc4eb1ba728)

**Comment by jilladams** [comment]
This was not reported on in Sprint 86. Adding to Sprint 87 for reporting purposes since it's now closed out.
